qttime.cc

```diff
@@ -0,0 +1,20 @@
+#include "qttime.h"
+
+int time_get_time(lua_State *L) {
```

Comment:
@AByzhynar, it should be renamed to qtime_get_time, or even qdatetime_get_datetime, to be more correct. At first should be library name, then function name.